ocs_ci/helpers/helpers.py

```diff
@@ -2003,6 +2052,42 @@ def create_pods_parallel(
     return pod_objs
 
 
+def delete_multiple_pvcs(pvc_objs):
```

Comment:
Can we use this function? https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/helpers/helpers.py#L2006 
This will delete all the objects in parallel

Reply:
This does not do in paralell, but does in thread which is not exacly what we want, in my function i delete them at once and its more accurate